core/src/main/java/com/netease/arctic/scan/BasicArcticFileScanTask.java

```diff
@@ -58,7 +58,7 @@ public BasicArcticFileScanTask(
     this.posDeleteFiles = posDeleteFiles == null ? Collections.emptyList() :
         posDeleteFiles.stream().filter(s -> {
           DataTreeNode node = FileNameGenerator.parseFileNodeFromFileName(s.path().toString());
-          return node.index() == baseFile.node().index() && node.mask() == baseFile.node().mask();
+          return baseFile.node().isSonOf(node) || baseFile.node().equals(node);
         }).collect(Collectors.toList());
```

Comment:
I can't understand the logical, plz make a simple description

Reply:
position-delete must be data file parent node or the same node